CONTRIBUTING.md

```diff
@@ -154,10 +173,23 @@ with [Discipline](https://github.com/typelevel/discipline) for law checking, and
  rely heavily on serialization, such as `Spark`, will have strong compatibility with `cats`.
  - Note that custom serialization tests are not required for instances of type classes which come from
  `algebra`, such as `Monoid`, because the `algebra` laws include a test for serialization.
+- For testing your laws, it is advised to check [this guide](https://typelevel.org/cats/typeclasses/lawtesting.html).
 
-TODO
+### Binary compatibility
+
+It is important to verify that the feature you are implementing is compatible with Scala 2.11.x and Scala 2.12.x (Scala <2.10.x is not supported). When you submit a PR, Travis makes this check, but it is time-expensive, so you can assure this step beforehand by issuing the command `+2.11.12`, which sets the cats' Scala version to `2.11.12` and then compile the project.
+
+A summary of these steps is as follows:
+
+```
+$ sbt
+> +2.11.12
+> compile
+```
+If you see errors in the compilation process related to the Scala version, you can execute the command `mimaReportBinaryIssues` to see what happened.
```

Comment:
binary breaking changes won't necessarily cause compilation errors.  So, instead of `compile` run `mimaReportBinaryIssues` should report the binary breaking issues.